crates/Makefile

```diff
@@ -10,7 +10,7 @@ fmt:  ## Run rustfmt and dprint
 
 .PHONY: check
 check:  ## Run cargo check with all features
-	cargo check -p polars --all-features
+	cargo check -p polars --all-features -p polars-stream
```

Comment:
Was this supposed to be part of the PR?

Reply:
Maybe.. Then we can ensure that all warnings are fixed. But if CI doesn't like it, I will remove it.